builder/release_image.go

```diff
@@ -275,6 +276,12 @@ func (j releaseBuildJob) Run() {
 		if err != nil {
 			return err
 		}
+
+		if r.ShowImageName {
+			r.UI.Printf("Image Name: %s\n", color.YellowString(imageName))
+			return nil
```

Comment:
Instead of returning here, I'd add another flag `--no-run` and then return `nil` if it is set after this `condition`.